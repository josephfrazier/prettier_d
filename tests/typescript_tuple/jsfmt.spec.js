run_spec(__dirname, ["typescript"]);
run_spec(__dirname, ["typescript"], { trailingComma: "es5" });
run_spec(__dirname, ["typescript"], { trailingComma: "all" });
