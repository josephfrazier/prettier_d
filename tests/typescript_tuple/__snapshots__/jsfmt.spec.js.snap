// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trailing-comma.ts 1`] = `
====================================options=====================================
parsers: ["typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
export interface ShopQueryResult {
  chic: boolean;
  location: number[];
  menus: Menu[];
  openingDays: number[];
  closingDays: [
    {
      from: string,
      to: string,
    }, // <== this one
  ];
  shop: string;
  distance: number;
}

=====================================output=====================================
export interface ShopQueryResult {
  chic: boolean;
  location: number[];
  menus: Menu[];
  openingDays: number[];
  closingDays: [
    {
      from: string;
      to: string;
    } // <== this one
  ];
  shop: string;
  distance: number;
}

================================================================================
`;

exports[`tuple.ts 1`] = `
====================================options=====================================
parsers: ["typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================

export type SCMRawResource = [
	number /*handle*/,
	string /*resourceUri*/,
	modes.Command /*command*/,
	string[] /*icons: light, dark*/,
	boolean /*strike through*/,
	boolean /*faded*/
];

=====================================output=====================================
export type SCMRawResource = [
  number /*handle*/,
  string /*resourceUri*/,
  modes.Command /*command*/,
  string[] /*icons: light, dark*/,
  boolean /*strike through*/,
  boolean /*faded*/
];

================================================================================
`;
